static/ankersrv.css

```diff
@@ -117,3 +117,26 @@ code {
     border: 0 none;
     padding: 0;
 }
+
+.fullscreen-emulate {
+    position: fixed;
+    width: 90vw;
+    height: 99vh;
+    top: 1%;
+    left: 5%;
```

Comment:
These "almost the whole way" values seem a little arbitrary. What are we trying to do here?

Why not `100%`?

And why also `max-width` / `max-height`?

Reply:
Because using 100vw/vh caused the content to somehow be larger than 100% of the viewport and thus caused scrollbars and the video to be too big. 

I tweaked around with different settings in a few different viewport sizes and found that these settings produced the same visual experience across all the viewports. 

The max-width and max-height were set to override the @media default css which was restricting the max size to 1322px for some reason.